library/core/src/ffi/c_str.rs

```diff
@@ -197,8 +197,8 @@ impl CStr {
     ///
     /// This function will wrap the provided `ptr` with a `CStr` wrapper, which
     /// allows inspection and interoperation of non-owned C strings. The total
-    /// size of the raw C string must be smaller than `isize::MAX` **bytes**
-    /// in memory due to calling the `slice::from_raw_parts` function.
+    /// size of the raw C string must be smaller than [`isize::MAX`] **bytes**
```

Comment:
The phrase "raw C string" feels weird to me, I don't feel like this documentation adequately describes what it means. That's preexisting though so you don't have to fix it, just noticing.